libs/langchain/langchain/chat_models/openai.py

```diff
@@ -320,9 +320,8 @@ def _generate(
         run_manager: Optional[CallbackManagerForLLMRun] = None,
         **kwargs: Any,
     ) -> ChatResult:
-        message_dicts, params = self._create_message_dicts(messages, stop)
-        params = {**params, **kwargs}
-        if self.streaming:
```

Comment:
wait why are we ignoring self.streaming

Reply:
> wait why are we ignoring self.streaming

`self.streaming` has be used to initialize `params["stream"]` in `self._default_params`.

`self._create_message_dicts` -> `self._client_params` -> `self._default_params`

Reply:
ah good call! should we del L327 then?

Reply:
Maybe it's useless, except making sure the stream is a boolean.